code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -514,8 +514,81 @@
 	if(M.mind?.assigned_role != "Clown")
 		REMOVE_TRAIT(M, TRAIT_COMIC_SANS, id)
 		M.RemoveElement(/datum/element/waddling)
+		M.DeleteComponent(/datum/component/squeak)
 
-	M.DeleteComponent(/datum/component/squeak)
+/datum/reagent/mimestrogen
+	name = "Mimestrogen"
+	id = "mimestrogen"
+	description = "Mimestrogen is an odd chemical compound that induces a variety of annoying side-effects in the average person. It also causes mild intoxication, and is toxic to clowns."
+	color = "#353535" // Should be dark grey, there are already a fair number of white chemicals
+	process_flags = ORGANIC | SYNTHETIC
+	drink_desc = "The color of the glass' surroundings seem to drain as you look at it."
+	taste_description = "an entertaining flavour"
+
+/datum/reagent/mimestrogen/on_new()
+	..()
+	var/mob/living/carbon/C = holder.my_atom
+	if(!istype(C))
+		return
+	if(C.mind)
+		if(C.mind.assigned_role == "Mime")
+			to_chat(C, "<span class='notice'>Whatever that was, it feels great!</span>")
+		else if(C.mind.assigned_role == "Clown")
+			to_chat(C, "<span class='warning'>You feel nauseous.</span>")
+			C.AdjustDizzy(volume STATUS_EFFECT_CONSTANT)
+			C.mind.miming = TRUE
+			ADD_TRAIT(C, TRAIT_COLORBLIND, id)
+		else
+			to_chat(C, "<span class='warning'>Something doesn't feel right...</span>")
+			C.AdjustDizzy(volume STATUS_EFFECT_CONSTANT)
+			C.mind.miming = TRUE // Jestosterone gives comic sans which makes one more clown-like, comic sans also unlocks clown healing, minus Jestoserone. So, mind.miming makes one more like a mime and unlocks mime healing, minus Mimestrogen.
+			ADD_TRAIT(C, TRAIT_COLORBLIND, id)
+
+/datum/reagent/mimestrogen/on_mob_life(mob/living/carbon/human/M)
+	var/update_flags = STATUS_UPDATE_NONE
+	if(prob(10))
+		M.emote("giggle")
+	if(!M.mind)
+		return ..() | update_flags
+	if(M.mind.assigned_role == "Mime")
+		update_flags |= M.adjustBruteLoss(-1.5 * REAGENTS_EFFECT_MULTIPLIER, robotic = TRUE)
+	else
+		M.AdjustDizzy(20 SECONDS, 0, 100 SECONDS)
+		if(M.client)
+			M.client.color = MATRIX_GREYSCALE
+			M.update_client_colour() // TRAIT_COLORBLIND only makes you colourblind for the wires, this fully makes it greyscale
+		if(prob(10))
+			M.EyeBlurry(10 SECONDS)
+		if(prob(6))
+			var/static/list/mime_message = list("You feel light-headed.",
+			"You can't see straight.",
+			"You feel about as entertaining as the station mime.",
+			"Muted colors and berets cloud your vision.",
+			"Your voice box feels numb.",
+			"What was that?!",
+			"You can hear silence in the distance, somehow.",
+			"You feel like miming.",
+			"Silence permeates your ears.",
+			"...",
+			"You feel like miming a performance.")
```

Comment:
PTAL with the style guide in mind! https://devdocs.paradisestation.org/coding/style_guidelines/#multi-line-procs-and-list-formatting